networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -528,6 +530,18 @@ def get_connected_devices(self, switches: List[NbRecord]) -> Tuple[Set[NbRecord]
                 # Those that we find unbundled are probably not modelled correctly, so we exclude them.
                 if not iface.lag:
                     continue
+                m = self.lag_id_re.match(iface.lag.name)
+                if not m:
+                    print(f'Cannot parse LAG id on interface {switch.name}/{iface.name}, '
+                          f'bundled in {iface.lag.name}')
+                lag_id = int(m.group('lag_id'))
+
+                if not RESPONSIBLE_LAG_RANGES[0] < lag_id < RESPONSIBLE_LAG_RANGES[1]:
```

Comment:
So the lag ranges are exclusive on both ends? I'd expect them to be inclusive on both ends, as with most (if not all) our other ranges in this piece of software.

Reply:
They indeed are exclusive. While technically our range is 100-199, inclusive, we will never allow a LAG of ID 100 as per our LAG ID derivation that LAG would need its first interface on interface index 0, which does not exists. Hence the range is exclusive.

Reply:
What do you think about making them inclusive and changing the range to `(101, 199)`?